lib/next_ls/updater.ex

```diff
@@ -65,6 +65,12 @@ defmodule NextLS.Updater do
           end
         end
 
+      {:ok, %{status: 403, body: %{"message" => msg}} = _body} ->
```

Comment:
Instead of adding these two specific checks, let's make the existing check less specific (don't pattern match on the error atom). 

Keep the status 200 in the first check tho

Reply:
Sure, no problem. By "error atom" do you mean the `status` field, or the `"message"` field? (I wasn't sure if Github returns a `message` for non-403 responses.)

Reply:
No, I mean the atom on line 74

Reply:
Done.